fromURL.go

```diff
@@ -127,11 +128,13 @@ func (client *Client) imageURL(url string) (*GenericInfo, int, error) {
 	ii, status, err := client.GetGalleryImageInfo(id)
 	if err == nil && status < 400 {
 		ret.GImage = ii
-
-		return &ret, status, err
+		return &ret, status, fmt.Errorf("client.GetGalleryImageInfo:%w", err)
```

Comment:
`err` is `nil` here and the response status code does not indicate an error, so we should return a `nil` error here.

```suggestion
		return &ret, status, nil
```

Reply:
good catch, fixed.